frontend/src/components/application/OpportunityCard.tsx

```diff
@@ -0,0 +1,214 @@
+    } else {
+      agencyTitle = agencyCode;
+    }
+
+    return agencyTitle;
+  };
+
+  const displayPostedDate = (postDate: string | null) => {
+    if (postDate) {
+      return getConfiguredDayJs()(postDate).format("MMM D, YYYY hh:mm A z");
+    }
+    return null;
+  };
+
+  const displayEstimatedAwardDate = (estimatedAwardDate: string | null) => {
+    if (estimatedAwardDate) {
+      return getConfiguredDayJs()(estimatedAwardDate).format(
+        "MMM D, YYYY hh:mm A z",
+      );
+    }
+    return null;
+  };
+
+  return (
+    <>
+      <Grid tablet={{ col: 6 }} mobile={{ col: 12 }}>
+        <dl className="margin-top-0">
+          <OpportunityItemLink
+            opKey={t("name")}
+            opValue={opportunity_title}
+            opId={opportunity_id}
+          />
+          <OpportunityItem opKey={t("number")} opValue={opportunity_number} />
+          <OpportunityItem
+            opKey={t("posted")}
+            opValue={displayPostedDate(post_date)}
+          />
+          <OpportunityItem
+            opKey={t("agency")}
+            opValue={displayAgencyAndCode(agency_name, agency_code)}
+          />
+          <OpportunityItem
+            opKey={t("assistanceListings")}
+            opValue={displayOpportunityAssistanceListings(
+              opportunity_assistance_listings,
+            )}
+          />
+          <OpportunityItem
+            opKey={t("costSharingOrMatchingRequirement")}
+            opValue={displayIsCostSharing(is_cost_sharing)}
+          />
+          <OpportunityItem
+            opKey={t("grantorContactInfomation")}
+            opValue={agency_contact_description}
+          />
+        </dl>
+      </Grid>
+      <Grid tablet={{ col: 6 }} mobile={{ col: 12 }}>
+        <h4 className="font-ui-md text-bold">{t("award")}</h4>
+        <dl className="margin-top-0">
+          <OpportunityItem
+            opKey={t("programFunding")}
+            opValue={formatCurrency(estimated_total_program_funding)}
+          />
+          <OpportunityItem
+            opKey={t("expectedAward")}
+            opValue={
+              expected_number_of_awards
+                ? String(expected_number_of_awards)
+                : null
+            }
+          />
+          <OpportunityItem
+            opKey={t("awardMaximum")}
+            opValue={formatCurrency(award_ceiling)}
+          />
+          <OpportunityItem
+            opKey={t("awardMinimum")}
+            opValue={formatCurrency(award_floor)}
+          />
+          <OpportunityItem
+            opKey={t("estimatedAwardDate")}
+            opValue={displayEstimatedAwardDate(close_date)}
+          />
+        </dl>
+      </Grid>
+    </>
+  );
+};
+
+export const OpportunityCard = (
+  opportunityOverview: OpportunityOverviewType,
+) => {
+  const t = useTranslations("Application.opportunityOverview");
+
+  return (
+    <GridContainer
+      data-testid="oppotunity-card"
+      className="padding-x-2 border border-gray-10"
```

Comment:
I imagine there might be a theme color we should be using here - the figma is a little unclear since #E6E6E2 is not in our palette, but my guess is this should be `border-primary`. Could check with @snyder-ux